recipyCmd/recipycmd.py

```diff
@@ -141,11 +142,9 @@ def main():
 
 def annotate(args):
     # check whether editor is set
-
-    if get_editor() is None:
-        print('No custom editor set in .recipyrc file.')
-        print('Could not find default $EDITOR - exiting.')
-        return
+    editor = get_editor()
+    if editor is None:
+        raise RuntimeError('No custom editor set in ~/.recipy/recipyrc file and could not find default $EDITOR.')
```

Comment:
Is this ever going to get run? If editor is None then `RuntimeError` will have already been raised in `get_editor` won't it? (I may be wrong...)